aws/data_source_aws_waf_web_acl.go

```diff
@@ -0,0 +1,64 @@
+package aws
```

Comment:
Can you please name this file `aws/data_source_aws_wafregional_web_acl.go` to match the data source naming? Thanks!